core/src/loader.rs

```diff
@@ -1576,6 +1576,11 @@ impl<'gc> Loader<'gc> {
                             tracing::warn!("Invalid URLLoaderDataFormat: {}", data_format);
                         }
 
+                        // strips off an optional UTF8 BOM at the beginning
```

Comment:
`bytearray::to_string` covers UTF-16 BOMs, and we'd probably like to cover them too. Additionally, it seems that this also applies to url variables

Reply:
done